test/v1/test_account_info.py

```diff
@@ -292,11 +292,39 @@ def test_upgrade_2_default_app_key(self):
     def _make_info(self):
         return self._make_sqlite_account_info()
 
-    def _make_sqlite_account_info(self, last_upgrade_to_run=None):
+    def _make_sqlite_account_info(self, env=None, last_upgrade_to_run=None):
         """
-        Returns a new StoredAccountInfo that has just read the data from the file.
+        Returns a new SqliteAccountInfo that has just read the data from the file.
         """
-        return SqliteAccountInfo(file_name=self.db_path, last_upgrade_to_run=None)
+        return SqliteAccountInfo(
+            file_name=self.db_path if not env else None,
+            env=env or {},
+            last_upgrade_to_run=last_upgrade_to_run
+        )
 
     def test_account_info_persistence(self):
         self._test_account_info(check_persistence=True)
+
+    def test_uses_xdg_config_home(self):
+        with tempfile.TemporaryDirectory() as d:
```

Comment:
`tempfile.TemporaryDirectory` was added in 3.2, but we still support 2.7, but we do have [b2sdk.utils.TempDir](https://github.com/Backblaze/b2-sdk-python/blob/43b42e7562ec82c8baefd2614b1be6aefbfb0e0e/b2sdk/utils.py#L239) context manager. Please switch to it.